util/src/main/java/com/netcracker/ConsoleHelper.java

```diff
@@ -0,0 +1,39 @@
+package com.netcracker;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class ConsoleHelper {
```

Comment:
Some kind of logging issues, it will be invoke when we need to read or to write something into console, so it is useful for server, also it will be useful for console client. Of course, I can use Log4 or something like this, but in my case, in this class all other classes should transmit console messages